maltoolbox/model.py

```diff
@@ -85,15 +85,17 @@ def add_asset(
                 if allow_duplicate_names:
                     asset.name = asset.name + ':' + str(asset.id)
                 else:
-                    raise ValueError(f'Asset name {asset.name} is a '
-                    'duplicate and we do not allow duplicates.')
+                    raise ValueError(
+                        f'Asset name {asset.name} is a duplicate'
```

Comment:
Still f-string.

Reply:
I focused on log-statements in this PR, not exceptions. I didn't get any warnings from my linter for f-strings in exceptions, so I let them be.

Reply:
oh right, false alarm.